libraries/resource_firewall.rb

```diff
@@ -19,5 +19,11 @@ class Resource::Firewall < Chef::Resource::LWRPBase
 
     # for firewall implementations where ipv6 can be skipped (currently iptables-specific)
     attribute(:ipv6_enabled, kind_of: [TrueClass, FalseClass], default: true)
+
+    if Chef::Platform.windows?
```

Comment:
I don't think we should guard this. I'd rather it just be a no-op than a compilation error.